dotnetcore-runtime.portable/tools/data.ps1

```diff
@@ -1,10 +1,10 @@
 ﻿@{
     PackageName = 'dotnetcore-runtime.portable'
-    Version = '2.0.3'
-    Url = 'https://download.microsoft.com/download/D/7/2/D725E47F-A4F1-4285-8935-A91AE2FCC06A/dotnet-sdk-2.0.3-win-x86.zip'
-    Checksum = '37deddbfc1a8ea72da4e864343bb4374bd50d744e1621910de30f8d83eaa3860b14f262750810a7ff66f8ab05a587cd71a935a130acf499257d8d4509e0b3cb2'
+    Version = '2.0.4'
+    Url = 'https://download.microsoft.com/download/2/B/2/2B2854E7-7EAE-4FE9-85D2-19ACCD716F18/dotnet-runtime-2.0.4-win-x86.zip'
```

Comment:
This is different from what it was but I suppose it is better?

Reply:
I had a look through the history, and it looks like I accidentally added the SDK previously instead of _just_ the runtime. This puts it back to the runtime only.